scripts/ci/unit-tests.sh

```diff
@@ -7,4 +7,3 @@
 set -ev
 
 scripts/ci/build.sh
-CHROME_BIN='/usr/bin/google-chrome-stable' yarn test
```

Comment:
Wait, I think you only wanted to delete the `CHROME_BIN` env var here... 😅

Reply:
naaaaaaaaaahhhhhh....

(oops)